src/lavinmq/http/controller/parameters.cr

```diff
@@ -12,12 +12,25 @@ module LavinMQ
       def details_tuple
         {
           name:      @p.parameter_name,
-          value:     @p.value,
+          value:     mask_secrets(@p.value),
           component: @p.component_name,
           vhost:     @vhost,
         }
       end
 
+      # Mask sensitive fields in parameter values (keys ending in -secret or -password)
+      private def mask_secrets(value : JSON::Any) : JSON::Any
+        return value unless h = value.as_h?
+        masked = h.transform_values do |v, k|
+          if (k.ends_with?("-secret") || k.ends_with?("-password")) && v.as_s?
+            JSON::Any.new("********")
```

Comment:
This makes sense for a human, but for an automated API it's less clear that is masked. Return nil instead?

Reply:
I think nil would easily be interpreted as "not set". Another though I had was to turn it into a bool, so return true is set false if not, but I found comfort in mimicking github :)